### ISSUE TITLE
`direct_mqmlqm` regression test fails

### ISSUE THREAD

**Issue by zouharvi** [AppraiseDev/Appraise#130]
```
RegressionTests/run.sh
```

The diff has some issues with IDs.

```
< engdeu0f01,ende-tutorial,0,TGT,eng,deu,10,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,ende-tutorial,1,TGT,eng,deu,20,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,ende-tutorial,2,TGT,eng,deu,30,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,ende-tutorial,3,TGT,eng,deu,40,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,ende-tutorial,4,TGT,eng,deu,50,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,wmt23.ONLINE-Y,5,TGT,eng,deu,60,guardian.114833,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,wmt23.refA,6,TGT,eng,deu,70,cnn.151755,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,wmt23.refA,7,TGT,eng,deu,80,cnbc.com.24647,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
< engdeu0f01,wmt23.refA,8,TGT,eng,deu,90,bbc.124285,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
---
> engdeu0f01,ende-tutorial,1000001,TGT,eng,deu,10,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,ende-tutorial,1000002,TGT,eng,deu,20,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,ende-tutorial,1000003,TGT,eng,deu,30,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,ende-tutorial,1000004,TGT,eng,deu,40,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,ende-tutorial,1000005,TGT,eng,deu,50,ende-tutorial,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,wmt23.refA,43,TGT,eng,deu,60,guardian.114833,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,wmt23.ONLINE-Y,31,TGT,eng,deu,70,cnn.151755,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,wmt23.ONLINE-Y,22,TGT,eng,deu,80,cnbc.com.24647,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
> engdeu0f01,wmt23.ONLINE-Y,7,TGT,eng,deu,90,bbc.124285,False,"{'start_i': 0, 'end_i': 50, 'severity': 'major'}"
```

This could be regression from the [data source](https://github.com/zouharvi/lightweight-quality-metrics/).

**Comment by zouharvi** [comment]
c7d91f0 reintroduced test fail

**Comment by zouharvi** [comment]
The error only appears if another campaign is already present. Likely not filtered enough.

**Comment by zouharvi** [comment]
Ok, so `item.id` is actually `item.itemID + item.taskID`... Fixed it in 0c1cbde.
